src/summary/documentation/vital-card-level-two.component.tsx

```diff
@@ -0,0 +1,185 @@
+                return (
+                  <React.Fragment key={vitals.id}>
+                    <tr>
+                      <td>{formatDate(vitals.date)}</td>
+                      <td>
+                        {`${vitals.systolic} / ${vitals.diastolic}`}
+                        {index === 0 && <span> mmHg </span>}
+                      </td>
+                      <td>
+                        {vitals.pulse} {index === 0 && <span>bpm</span>}
+                      </td>
+                      <td>
+                        {vitals.oxygenation} {index === 0 && <span>%</span>}
+                      </td>
+                      <td>
+                        {vitals.temperature}
+                        {index === 0 && <span> &#8451; </span>}
+                      </td>
+                      <td>
+                        <svg className="omrs-icon" fill="rgba(0, 0, 0, 0.54)">
+                          <use xlinkHref="#omrs-icon-chevron-right" />
+                        </svg>
+                      </td>
+                    </tr>
+                  </React.Fragment>
+                );
+              })}
+          </tbody>
+        </table>
+        <div className={styles.pagination}>
+          <div>
+            {showPreviousButton && (
+              <button
+                onClick={previousPage}
+                className={`${styles.navButton} omrs-bold omrs-btn omrs-text-neutral omrs-rounded`}
+              >
+                <svg className="omrs-icon" fill="rgba(0, 0, 0, 0.54)">
+                  <use xlinkHref="#omrs-icon-chevron-left" />
+                </svg>
+                <Trans i18nKey="Previous">Previous</Trans>
+              </button>
+            )}
+          </div>
+          <div>
+            <Trans i18nKey="Page">Page</Trans> {currentPage}{" "}
+            <Trans i18nKey="of">of</Trans> {totalPages}
+          </div>
+          <div>
+            {showNextButton && (
+              <button
+                onClick={nextPage}
+                className={`${styles.navButton} omrs-bold omrs-btn omrs-text-neutral omrs-rounded`}
+              >
+                <Trans i18nKey="Next">Next</Trans>
+                <svg className="omrs-icon" fill="rgba(0, 0, 0, 0.54)">
+                  <use xlinkHref="#omrs-icon-chevron-right" />
+                </svg>
+              </button>
+            )}
+          </div>
+        </div>
+      </SummaryCard>
+    );
+  }
+  function displayNoPatientsVitals() {
+    return (
+      <SummaryCard name="Vitals" match={props.match} styles={{ width: "100%" }}>
+        <div className={`${styles.vitalsAbsent} omrs-bold`}>
+          <p>
+            <Trans i18nKey="NoVitalsDocumentated">
+              No Vitals are documentated
+            </Trans>
+          </p>
+          <Trans i18nKey="Please">Please</Trans>{" "}
+          <a href="/">
+            {" "}
+            <Trans i18nKey="AddPatientVitals">add patient vitals</Trans>
```

Comment:
Are there keys showing up in the en.json file in the translations directory? I don't see that file as part of this commit.

Reply:
I was playing around with this on the login component. It seems that the babel plugin doesn't recognize the variable. This means the key to be translated doesn't get inserted automatically in en.json file. You can add the key manually but I noticed that the babel plugin removes the manually inserted keys (probably because it removes anything that's not named in the current file). As a solution, we could set the  discardOldKeys to false at least for those modules in which you have the need to pass the key as a prop. Of course there are probably downsides to this with regards to key management but I think still worth it. 

In any event, this change **does not** belong in this PR anyways. Please don't worry about responding to these comments in this PR. We can handle in another PR.

Reply:
I took a look at your branch at https://github.com/donaldkibet/openmrs-esm-patient-chart/tree/MF-97

You don't have the babel plugin. See https://github.com/openmrs/openmrs-esm-login/blob/master/.babelrc to see how this is done. I'm not a babel expert by any means but I'm assuming you can just copy that text and paste into the .babelrc file for this module and everything should work.

Reply:
> Are there keys showing up in the en.json file in the translations directory? I don't see that file as part of this commit.

Had not included the keys, will address the translation part in a different PR